Makefile

```diff
@@ -0,0 +1,122 @@
+VERSION := 0.1+
+
+PACKAGE       = github.com/kata-containers/ksm-throttler
```

Comment:
The github URL is used in a lot of places. It's not a blocker for this PR of course, but I wonder if we could standardise on something like `.ci/self-url` that container this string and then have the `Makefile` read that file. `make` can then generate the value used in `ksm-throttler.service.in`, and so on.